junit5-engine/src/main/java/org/junit/gen5/engine/junit5/descriptor/MethodTestDescriptor.java

```diff
@@ -120,10 +114,7 @@ public JUnit5EngineExecutionContext prepare(JUnit5EngineExecutionContext context
 			context.getExecutionListener(), this, testInstance);
 
 		// @formatter:off
-		return context.extend()
-				.withExtensionRegistry(extensionRegistry)
-				.withExtensionContext(testExtensionContext)
-				.build();
+		return context.extend().withExtensionRegistry(extensionRegistry).withExtensionContext(testExtensionContext).build();
 		// @formatter:on
 	}
```

Comment:
I didn't do this intentionally. I'll check my IntelliJ formatter settings to see if they are misconfigured. Thanks.